bundles/net.tourbook.common/src/net/tourbook/common/color/GraphColorManager.java

```diff
@@ -59,7 +59,7 @@
    public static final String PREF_GRAPH_PACE                         = "pace";                       //$NON-NLS-1$
    public static final String PREF_GRAPH_POWER                        = "power";                      //$NON-NLS-1$
    public static final String PREF_GRAPH_SPEED                        = "speed";                      //$NON-NLS-1$
-   public static final String PREF_GRAPH_TEMPTERATURE                 = "tempterature";               //$NON-NLS-1$
+   public static final String            PREF_GRAPH_TEMPTERATURE                 = "temperature";                //$NON-NLS-1$
```

Comment:
If you fix the string, can you also fix the typo in the variable name ? PREF_GRAPH_TEMP**T**ERATURE

Reply:
@FJBDev  done (in my coming PR)